cmd/helm/root.go

```diff
@@ -67,6 +67,8 @@ Environment variables:
 | $HELM_KUBEASUSER                   | set the Username to impersonate for the operation.                                                |
 | $HELM_KUBECONTEXT                  | set the name of the kubeconfig context.                                                           |
 | $HELM_KUBETOKEN                    | set the Bearer KubeToken used for authentication.                                                 |
+| $HELM_KUBEINSECURE_SKIP_TLS_VERIFY | indicate if the kube api server's certificate validation is be skipped (insecure)                 |
+| $HELM_KUBETLS_SERVER_NAME          | set the server name used to validate the kube api server certificate                              |
```

Comment:
kube --> kubernetes

Reply:
resolved in 9898a890f971b20a3af1dd6792d258694a5f97f7